tests/attestation_cert_apis/enc/enc.cpp

```diff
@@ -236,35 +242,50 @@ oe_result_t get_tls_cert_signed_with_key(
         goto done;
     }
 
-    // validate cert with oe_verify_attestation_certificate_with_evidence()
+    // validate cert with oe_verify_attestation_certificate_with_evidence_v2()
     // to ensure that the added report verifier part of the function works well
-    result = oe_verify_attestation_certificate_with_evidence(
-        output_cert, output_cert_size, enclave_claims_verifier, nullptr);
+    result = oe_verify_attestation_certificate_with_evidence_v2(
+        output_certificate,
+        output_certificate_size,
+        nullptr,
+        0,
+        nullptr,
+        0,
+        &claims,
+        &claims_length);
+
     OE_TRACE_INFO(
         "\nFrom inside enclave: verifying the certificate with "
-        "oe_verify_attestation_certificate_with_evidence()... %s\n",
+        "oe_verify_attestation_certificate_with_evidence_v2()... %s\n",
+        result == OE_OK ? "Success" : "Fail");
```

Comment:
```suggestion
        oe_result_str(result));
```
I think we can output `oe_result` directly to acquire more info in failed.